packages/node/src/async/hooks.ts

```diff
@@ -50,3 +50,75 @@ export function setHooksAsyncContextStrategy(): void {
 
   setAsyncContextStrategy({ getCurrentHub, runWithAsyncContext });
 }
+/**
+ * Initiates the counting of created promises and settled promises.
+ *
+ * If `locations` is true, it will attempt to find the locations of each created promise
+ * and return an object whose keys are the callsites as strings and whose values are the
+ * number of promises created at that callsite.
+ *
+ * If `continuation` is true, promises will only be counted if there is an async
+ * continuation chain (as determined by AsyncLocalStorage) back to the given `startCounter()` call.
+ * This is helpful for filtering out unrelated promises, like ones from an unrelated concurrent HTTP request.
+ *
+ * @param {Object} opts - Options for counting promises.
+ * @param {boolean} [opts.locations=false] - Whether to count promise locations.
+ * @param {boolean} [opts.continuation=false] - Whether to consider async continuation chains.
+ * @returns {function(): number | { [key: string]: number }} - A function to get the count of created and settled promises.
+ */
+/**
+ *
+ */
```

Comment:
```suggestion
```

Reply:
Removing of these lines gives eslint error on line 72 function 

> Missing JSDoc comment.eslint[jsdoc/require-jsdoc](https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-require-jsdoc)